package.json

```diff
@@ -43,12 +43,15 @@
     "babel-eslint": "4.1.5",
     "babel-loader": "5.3.2",
     "bootstrap-sass": "3.3.6",
+    "chai": "3.5.0",
+    "chai-as-promised": "5.2.0",
     "escope": "3.3.0",
     "eslint": "1.9.0",
     "eslint-config-airbnb": "2.0.0",
     "eslint-plugin-no-console-log": "1.0.0",
     "eslint-plugin-react": "3.9.0",
     "gulp": "3.9.0",
+    "gulp-autoprefixer": "^3.1.0",
```

Comment:
Carets make the build non-deterministic (trademarked by sam) so just take it out and we will manually up the version if we ever need to.